dockerfiles/artifactory/Dockerfile

```diff
@@ -1,19 +1,33 @@
-FROM node:12-buster-slim
+FROM snyk/ubuntu as base
 
 MAINTAINER Snyk Ltd
 
+USER root 
+
+RUN apt update && apt install -y make python gcc
+
+USER node 
+
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+
+ENV PATH=$PATH:/home/node/.npm-global/bin 
+
 RUN npm install --global snyk-broker
 
+USER root 
+
 RUN apt-get update && apt-get install -y ca-certificates
```

Comment:
Just want to verify that this is done by purpose. We have `RUN apt update` on line 7. Can we make certificate installation there as well?

Reply:
This is on purpose, as this is the idea of using a [multistage build in docker](https://docs.docker.com/develop/develop-images/multistage-build/).
But this question did help me think that I accidentally put the certs installation before the second phase (`FROM`), when it should be after, so that it'll be in the final container, so nice catch anyway.